pkg/util/sys/device.go

```diff
@@ -16,6 +16,7 @@ limitations under the License.
 package sys
 
```

Comment:
Are you going to revert this file? Looks like the changes all moved to the other package per a previous suggestion, right?

Reply:
The only modification that is present in this file is the `CephVolumeDeviceData` struct declaration, and the addition of a new field in the `LocalDisk` struct to store a pointer to `CephVolumeDeviceData`.

I think that is better to keep together this two structs. 

I have used a pointer in `LocalDisk`  to make json marshaling able to  completely remove the `cephVolumeData`  field in the configmaps when the discovery daemon runs for other backends different from Ceph.